test/solvers/advanced/proximal_operator_test.py

```diff
@@ -230,7 +230,7 @@ def test_proximal_convconj_l2_sq_wo_data():
     x = space.element(x_arr)
 
     # Create data
-    g = space.element(-2 * x_arr)
+    g = space.zero()
```

Comment:
If it is w.o. data, you are indeed correct, but then this should be removed. The reason we have this differently set up is because we know the function does optimization depending on if it is given `g` or not. By setting it to 0 instead of `None` we don't get to test these optimizations.